at_onboarding_cli/lib/src/at_onboarding_preference.dart

```diff
@@ -0,0 +1,10 @@
+import 'dart:core';
```

Comment:
This import is not necessary. Cause I see String is the only data type that comes from dart sdk. We don't need it. Please LMK if something goes wrong if we remove it.